tests/test_idna_codec.py

```diff
@@ -26,7 +26,7 @@ def testIncrementalDecoder(self):
             self.assertEqual("".join(codecs.iterdecode((bytes([c]) for c in encoded), "idna")),
```

Comment:
Is this part still necessary? `idna` does not appear to have a `iterdecode` method.